.travis.yml

```diff
@@ -1,3 +1,38 @@
 language: generic
 
-script: echo 'testing from .travis.yml'
+env:
+  DOCKER_COMPOSE_VERSION: 1.4.2
+
+services:
+  - docker
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+script:
+  - docker-compose build
+  - docker-compose run api ./docker-travis.sh
```

Comment:
Eventually, this should be changed to `docker-compose run api go test`. Ok for now.